index.html

```diff
@@ -305,13 +305,13 @@ <h4 class="panel-title">
             </tr>
           </tbody>
           <tbody ng-repeat="bufferline in bufferlines">
-            <tr class="bufferline">
+            <tr class="bufferline" ng-class="::{'repeated-prefix': bufferline.prefixtext==bufferlines[$index-1].prefixtext, 'repeated-time': (((60*+bufferline.shortTime.substr(0, bufferline.shortTime.indexOf(':')) + +bufferline.shortTime.substr(bufferline.shortTime.indexOf(':')+1)) - (60*+bufferlines[$index-1].shortTime.substr(0, bufferlines[$index-1].shortTime.indexOf(':')) + +bufferlines[$index-1].shortTime.substr(bufferlines[$index-1].shortTime.indexOf(':')+1)) + 1440) % 1440) < 3, 'highlight': bufferline.highlight}">
               <td class="time">
-                <span class="date" ng-class="::{'repeated-time': bufferline.shortTime==bufferlines[$index-1].shortTime}">
+                <span class="date">
                   <span class="cof-chat_time cob-chat_time coa-chat_time" ng-bind-html="::bufferline.formattedTime"></span>
                 </span>
               </td>
-              <td class="prefix"><span ng-class="::{'repeated-prefix': bufferline.prefixtext==bufferlines[$index-1].prefixtext}"><a ng-click="addMention(bufferline.prefix)"><span class="hidden-bracket">&lt;</span><span ng-repeat="part in ::bufferline.prefix" ng-class="::part.classes" ng-bind="::part.text|prefixlimit:25"></span><span class="hidden-bracket">&gt;</span></a></span></td><!--
+              <td class="prefix"><span><a ng-click="addMention(bufferline.prefix)"><span class="hidden-bracket">&lt;</span><span ng-repeat="part in ::bufferline.prefix" ng-class="::part.classes" ng-bind="::part.text|prefixlimit:25"></span><span class="hidden-bracket">&gt;</span></a></span></td><!--
```

Comment:
Does the outermost `span` have any function left? I guess we could just style the `a` it contains instead